perses/rjmc/geometry.py

```diff
@@ -1866,7 +1867,7 @@ def __init__(self, reference_system, torsion_proposal_order, global_parameter_na
 
```

Comment:
here is where we begin tallying `omitted_terms` with reference to a `reference_connectivity_graph`